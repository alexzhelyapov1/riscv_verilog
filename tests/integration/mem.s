.section .text
.global _start

_start:
    addi x1, x0, 7
    sd x1, 0x30(x0)
    ld x2, 0x30(x0)
    nop
    nop
    nop
    nop
    nop
    nop
    nop
    nop
    nop
    nop
    nop
    nop
    nop
    nop
    nop
    nop
    nop