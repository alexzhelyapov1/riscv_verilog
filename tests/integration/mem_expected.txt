x
x
0000000000000007
x
0000000000000007
0000000000000000
0000000000000000
0000000000000000
0000000000000000
0000000000000000
0000000000000000
0000000000000000